tests/kafkatest/services/mirror_maker.py

```diff
@@ -185,14 +185,14 @@ def start_node(self, node):
         self.logger.debug("Mirror maker is alive")
 
     def stop_node(self, node, clean_shutdown=True):
-        node.account.kill_process("java", allow_fail=True, clean_shutdown=clean_shutdown)
+        node.account.kill_java_processes("MirrorMaker", allow_fail=True, clean_shutdown=clean_shutdown)
```

Comment:
In some other classes (e.g. verifiable_consumer.py), we implement a `java_class_name` function which provides the simple name of the java class being executed. Maybe we could standardize on that convention and use it here and other classes?

Reply:
I think that's a really good idea.  I modified a bunch of service classes to do that.  Hopefully it makes it clearer what's going on.  It should make it easier to patch if someone needs to change the class name used in a service, as well...